runtime-transaction/Cargo.toml

```diff
@@ -27,6 +27,7 @@ name = "solana_runtime_transaction"
 bincode = { workspace = true }
 criterion = { workspace = true }
 rand = { workspace = true }
+solana-stake-program = { workspace = true }
```

Comment:
This dependency is added specifically for a test. Can the test be reworked to move this dependency out of SVM related crates? We are trying to keep the deps to a minimum so that SVM can be moved to its own Github repo.

Reply:
thanks for pointing this out, change in 34f497d76ee8d0324ff731d3348a60223a6ef539